theme/mixins/backdrop.less

```diff
@@ -7,4 +7,5 @@
   background-color: @sk-black;
   opacity: 0.7;
   transition: opacity 0.3s;
+  z-index: @z-index-negative;
```

Comment:
I'm not feeling all that comfortable introducing this... any element on a page with an autosuggest with a z-index will sit higher than the backdrop right?  Perhaps another solution could be to put a z-index of 2 on the suggestions container and leave the z-index of 1 on the container?